src/main/java/jalau/at18/katas/likes/adriana/Main.java

```diff
@@ -0,0 +1,55 @@
+package jalau.at18.katas.likes.adriana;
+
+public class Main {
+    static final Integer MAX_NUM_NAMES = 3;
+    public static void main(String[] args) {
+        String[] array = {"Jose", "Maria", "Pedro", "Susana"};
+        printArray(array);
+        System.out.println(likesText(array));
+    }
+    public static void printArray(String[] array) {
+        Integer length = array.length;
+        System.out.print("[");
+        for (int count = 0; count < length; count++) {
+            if (count < length - 1) {
+                System.out.print(array[count] + ", ");
+            } else {
+                System.out.println(array[count] + "]");
+            }
+        }
+    }
+    public static String likesText(String[] str) {
+        Integer length = str.length;
+        String text = "";
+        if (length == 0) {
+            text = "no one";
+        } else {
+            for (int count = 0; count < length; count++) {
+                if (count == 0) {
+                    text = str[count];
+                } else {
+                    if (length <= MAX_NUM_NAMES) {
+                        if (count < length - 1) {
```

Comment:
Implementation is good, but something that I will suggest you to consider for your next implementation is to avoid too many nested blocks. Instead, consider simplify your code logic or create smaller methods that address smaller goals